addons/flannel/template/base/install.sh

```diff
@@ -34,7 +34,7 @@ function flannel() {
     flannel_render_config
 
     if flannel_weave_conflict; then
-        printf "${YELLOW}Would you like to migrate from Weave to FLannel?${NC}"
+        printf "${YELLOW}Would you like to migrate from Weave to Flannel?${NC}"
```

Comment:
Could we use logWarn here instead of `printf "${YELLOW}`?

Reply:
No, because this isn't a warning - it's a prompt